client/my-sites/site-settings/controller.js

```diff
@@ -85,32 +79,11 @@ const controller = {
 		}
 	},
 
-	siteSettings( context ) {
-		let analyticsPageTitle = 'Site Settings';
-		const basePath = route.sectionify( context.path );
-		const siteId = getSelectedSiteId( context.store.getState() );
-		const section = sectionify( context.path ).split( '/' )[ 2 ];
-		const state = context.store.getState();
-
-		// FIXME: Auto-converted from the Flux setTitle action. Please use <DocumentHead> instead.
-		context.store.dispatch( setTitle( i18n.translate( 'Site Settings', { textOnly: true } ) ) );
-
-		// if site loaded, but user cannot manage site, redirect
-		if ( siteId && ! canCurrentUser( state, siteId, 'manage_options' ) ) {
```

Comment:
Yes, the General section will now use the shared settings controller, and it has that check, see https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/site-settings/settings-controller.js#L27